sourcecode/hub/resources/views/admin/lti-tools/add.blade.php

```diff
@@ -51,8 +54,57 @@
             </label>
         </p>
 
+        <fieldset>
+            <legend>{{ trans('messages.edit-mode') }}</legend>
+
+            <div class="position-relative form-check">
+                <div class="form-check">
+                    <input
+                        type="radio"
+                        name="edit_mode"
+                        value="{{ LtiToolEditMode::Replace }}"
+                        aria-describedby="edit-mode-replace-help"
+                        class="form-check-input"
+                        id="edit-mode-replace"
+                        @checked(old('edit_mode', true))
+                    >
+                    <label
+                        class="form-check-label stretched-link"
+                        for="edit-mode-replace"
```

Comment:
The recommendation is to nest the input and label, putting the input inside the label, this attribute can then be removed